### PR TITLE
docs(*) consolidate T-shirt instructions

### PR DESCRIPTION
**Pull request by judithpatudith**
**NOTE**: Please read the CONTRIBUTING.md guidelines before submitting your patch,
and ensure you followed them all:

https://github.com/Kong/docs.konghq.com/blob/master/CONTRIBUTING.md

### Summary

Point the contributor t-shirt instructions to Kong repo to make them easier to maintain.

### Full changelog

* Replace text in the t-shirt section with basic guidelines and a pointer to Kong's contrb. file 

### Issues resolved

Fix #XXX

### Checklist:

- [x] [Commit message & atomicity](https://github.com/Kong/docs.konghq.com/blob/master/CONTRIBUTING.md#commit-atomicity) checked
- [ ] Documentation 
- [x] Spellchecked my updates
- [x] Ready to be merged

### CODE REVIEW

**Comment by judithpatudith** [comment]
[Edit t-shirt contributing sections to define contributions more tightly](https://trello.com/c/D6eqArXu/127-edit-t-shirt-contributing-sections-to-define-contributions-more-tightly)

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -376,15 +376,8 @@ will cause a broken ToC, and should be corrected to:
...
-[Kong/docs.konghq.com](https://github.com/Kong/docs.konghq.com) was accepted,
-congratulations, you are eligible to receive the very special Contributor
-T-shirt! Go ahead and fill-out the [Contributors Submissions
-form](https://goo.gl/forms/5w6mxLaE4tz2YM0L2).
-
-Proudly wear your T-shirt and show it to us by tagging
-[@thekonginc](https://twitter.com/thekonginc) on Twitter!
-
-![Kong Contributor T-shirt](https://konghq.com/wp-content/uploads/2018/04/100-contributor-t-shirt-1024x768.jpg)
+Contributions to this repo that fix a bugs, adds functionality, or make it
```
Comment (thibaultcha) [inline CONTRIBUTING.md]: typo: bug

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -376,15 +376,8 @@ will cause a broken ToC, and should be corrected to:
...
+significantly easier to understand or use Kong will earn you a Kong contributor
+t-shirt! Find out how to request your shirt [here](https://github.com/Kong/kong/blob/master/CONTRIBUTING.md#contributor-t-shirt).
```
Comment (thibaultcha) [inline CONTRIBUTING.md]: t-shirt: T-shirt

Comment (thibaultcha) [reply CONTRIBUTING.md]: I believe that the previous phrasing read better. Also, we got rid of the "mention us on Twitter" and the picture, but should probably keep them.

> If your Pull Request to Kong/docs.konghq.com was accepted and fixes a bug, adds functionality, or makes it significantly easier to use or understand Kong, congratulations! You are eligible to receive the very special Contributor T-shirt! Go ahead and fill-out the Contributors Submissions form.

Comment (judithpatudith) [reply CONTRIBUTING.md]: The idea here was to reduce the number of places to update if something about the shirt changed, (for example the design or what to say in the tweet). 

I'm ok with keeping a single source of truth in the Kong repo or duplicating the content, but if there are other repos that will eventually qualify for shirts (ingress controller, pod injector for service mesh, etc.) then we might end up duplicating things a lot of places, and will have more to maintain when the shirt design gets a refresh.

Comment (thibaultcha) [reply CONTRIBUTING.md]: I was wondering if that was the intent. In this case, we can replace "contributions to Kong/docs.konghq.com" with "contributions to this repo", and end up with no more than a single paragraph we can copy paste in each repository, but more readable than the previous version of this paragraph. How does that sound?

Comment (judithpatudith) [reply CONTRIBUTING.md]: Maybe? I think this is more a theoretical question about what is part of Kong. If contributing to a Kong supported plugin gets a shirt, then maintaining your own open source plugin should also get you a shirt too. Maybe any integration should get you one (terraform / puppet modules, other stuff...) Not sure where we want to draw the line.

Comment (james-callahan) [reply CONTRIBUTING.md]: >  If contributing to a Kong supported plugin gets a shirt, then maintaining your own open source plugin should also get you a shirt too.

I don't think so. The obvious line to draw is things included in the kong rockspec an external plugins: https://github.com/Kong/kong/blob/8ed77f7308d823b321ef713cea0868040f9257b9/kong-0.14.1-0.rockspec#L37
Though you *could* extend it if you wanted it to.

Comment (judithpatudith) [reply CONTRIBUTING.md]: I don't think its really up to me, it would probably be a Marco and Thibault discussion. Let's take it offline for now?

Comment (judithpatudith) [reply CONTRIBUTING.md]: For anyone following along at home, we will be including plugins when they are moved to a different repo.

**Comment by judithpatudith** [comment]
@thibaultcha I think the comments are all addressed and this is ready for you to have another look when you have time.

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -376,7 +376,7 @@ will cause a broken ToC, and should be corrected to:
 
 ### Contributor T-shirt
 
-If your Pull Request to Kong/docs.konghq.com was accepted and fixes a bug, adds
+If your contribution to this repo was accepted and fixes a bug, adds
```
Comment (james-callahan) [inline CONTRIBUTING.md]: I don't think saying "this repo" is right, we should enumerate or list criteria which repositories are considered acceptable.

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -376,15 +376,11 @@ will cause a broken ToC, and should be corrected to:
...
-
-Proudly wear your T-shirt and show it to us by tagging
-[@thekonginc](https://twitter.com/thekonginc) on Twitter!
-
-![Kong Contributor T-shirt](https://konghq.com/wp-content/uploads/2018/04/100-contributor-t-shirt-1024x768.jpg)
+If your contribution to this repo was accepted and fixes a bug, adds
+functionality, or makes it significantly easier to use or understand Kong,
+congratulations! You are eligible to receive the very special Contributor
+T-shirt! Find out how to request your shirt
+[here](https://github.com/Kong/kong/blob/master/CONTRIBUTING.md#contributor-t-shirt).
```
Comment (james-callahan) [inline CONTRIBUTING.md]: Doesn't this link just point back to this exact same section?

Comment (james-callahan) [reply CONTRIBUTING.md]: Nope, ignore me here.

File: CONTRIBUTING.md (hunk 3 above)
Comment (james-callahan) [reply CONTRIBUTING.md]: I got confused and thought this was in kong/kong instead of kong/docs.konghq.com. I think you can ignore this.

**Comment by thibaultcha** [comment]
@kikito I am currently working on this merge, thanks :)

**Comment by thibaultcha** [comment]
Merged! Thanks @judithpatudith

**Comment by thibaultcha** [comment]
@judithpatudith "Thank you for contributing! Be sure to grab your own [contributor T-shirt](https://github.com/Kong/kong/blob/master/CONTRIBUTING.md#contributor-t-shirt) which you more than deserve after your hard work, and wear it proudly!" :wink: 

I recommend everyone to save replies like this one in their GitHub replies templates :)

**Pull request closed without merging**